files/index.html

```diff
@@ -0,0 +1,8 @@
+<html>
+    <head>
+        <title>Welcome to Jediluke!</title>
+    </head>
+    <body>
+    	<h1>Coming soon! Check back later...<h1>
```

Comment:
This is just a small thing, but can we be consistent in how we lay out the HTML. It looks like we have 4 spaces indent above, so we should have the same here.

Actually I think we should use 3 spaces for indents everywhere in HTML